<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="0" total="7" passed="4">
  <reporter-output>
    <line>
      <![CDATA[<br>Verification failure : expected [Protractor practice website - Banking App] but found [Title]<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_25_IST_2020.jpg><img src=Thu_Dec_31_16_35_25_IST_2020.jpg height=200 width=200></img></a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[Reporter.log() - Login Test successfully executed]]>
    </line>
    <line>
      <![CDATA[Capturing Screenshot.......]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_29_IST_2020.jpg>Screenshot in new Window</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_29_IST_2020.jpg><img height=200 width=200 src=Thu_Dec_31_16_35_29_IST_2020.jpg>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="23548" started-at="2020-12-31T16:35:23 IST" finished-at="2020-12-31T16:35:47 IST">
    <groups>
    </groups>
    <test name="Bank Manager Login Test" duration-ms="6073" started-at="2020-12-31T16:35:23 IST" finished-at="2020-12-31T16:35:29 IST">
      <class name="com.learning.testcases.BankManagerLoginTest">
        <test-method status="FAIL" signature="bankManagerLoginTest()[pri:0, instance:com.learning.testcases.BankManagerLoginTest@306cf3ea]" name="bankManagerLoginTest" duration-ms="5138" started-at="2020-12-31T16:35:23 IST" finished-at="2020-12-31T16:35:28 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Bank Manager Login failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Bank Manager Login failed
at org.testng.Assert.fail(Assert.java:99)
at com.learning.testcases.BankManagerLoginTest.bankManagerLoginTest(BankManagerLoginTest.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br>Verification failure : expected [Protractor practice website - Banking App] but found [Title]<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_25_IST_2020.jpg><img src=Thu_Dec_31_16_35_25_IST_2020.jpg height=200 width=200></img></a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[Reporter.log() - Login Test successfully executed]]>
            </line>
            <line>
              <![CDATA[Capturing Screenshot.......]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_29_IST_2020.jpg>Screenshot in new Window</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Thu_Dec_31_16_35_29_IST_2020.jpg><img height=200 width=200 src=Thu_Dec_31_16_35_29_IST_2020.jpg>]]>
            </line>
          </reporter-output>
        </test-method> <!-- bankManagerLoginTest -->
      </class> <!-- com.learning.testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test name="Add Customer Test" duration-ms="8842" started-at="2020-12-31T16:35:30 IST" finished-at="2020-12-31T16:35:39 IST">
      <class name="com.learning.testcases.AddCustomerTest">
        <test-method status="PASS" signature="addCustomerTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.AddCustomerTest@49dc7102]" name="addCustomerTest" duration-ms="5264" started-at="2020-12-31T16:35:30 IST" data-provider="dp" finished-at="2020-12-31T16:35:35 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{postCode=500089.0, RunMode=Y, lastName=Verma, firstName=Rajat}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="SKIP" signature="addCustomerTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.AddCustomerTest@49dc7102]" name="addCustomerTest" duration-ms="0" started-at="2020-12-31T16:35:35 IST" data-provider="dp" finished-at="2020-12-31T16:35:35 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{postCode=125001.0, RunMode=N, lastName=Sharma, firstName=Shreya}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at com.learning.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="PASS" signature="addCustomerTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.AddCustomerTest@49dc7102]" name="addCustomerTest" duration-ms="3423" started-at="2020-12-31T16:35:35 IST" data-provider="dp" finished-at="2020-12-31T16:35:39 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{postCode=110009.0, RunMode=Y, lastName=Arora, firstName=Minakshi}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
      </class> <!-- com.learning.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
    <test name="Open Account Test" duration-ms="7884" started-at="2020-12-31T16:35:39 IST" finished-at="2020-12-31T16:35:47 IST">
      <class name="com.learning.testcases.OpenAccountTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.learning.testcases.OpenAccountTest@59af0466]" name="setup" is-config="true" duration-ms="40776" started-at="2020-12-31T16:34:42 IST" finished-at="2020-12-31T16:35:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="openAccountTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.OpenAccountTest@59af0466]" name="openAccountTest" duration-ms="4325" started-at="2020-12-31T16:35:39 IST" data-provider="dp" finished-at="2020-12-31T16:35:43 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{currency=Dollar, RunMode=Y, customer=Rajat}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method status="SKIP" signature="openAccountTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.OpenAccountTest@59af0466]" name="openAccountTest" duration-ms="0" started-at="2020-12-31T16:35:43 IST" data-provider="dp" finished-at="2020-12-31T16:35:43 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{currency=Rupee, RunMode=N, customer=Shreya}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the Run mode for data set is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the Run mode for data set is NO
at com.learning.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method status="PASS" signature="openAccountTest(java.util.Hashtable)[pri:0, instance:com.learning.testcases.OpenAccountTest@59af0466]" name="openAccountTest" duration-ms="3299" started-at="2020-12-31T16:35:43 IST" data-provider="dp" finished-at="2020-12-31T16:35:46 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{currency=Pound, RunMode=Y, customer=Minakshi}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.learning.testcases.OpenAccountTest@59af0466]" name="tearDown" is-config="true" duration-ms="2781" started-at="2020-12-31T16:35:47 IST" finished-at="2020-12-31T16:35:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.learning.testcases.OpenAccountTest -->
    </test> <!-- Open Account Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
